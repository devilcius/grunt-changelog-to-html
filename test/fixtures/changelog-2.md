## Test bullets

- first
- second
- third