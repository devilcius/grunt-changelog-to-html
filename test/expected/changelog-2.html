<h2 id="test-bullets">Test bullets</h2>
<ul>
<li>first</li>
<li>second</li>
<li>third</li>
</ul>
